src/vivarium/framework/results/manager.py

```diff
@@ -14,6 +14,9 @@
     from vivarium.framework.engine import Builder
 
 
+METRICS_COLUMN = "value"
```

Comment:
This is the main thing. Everything else in the PR (except for the __init__ change) is importing this variable and then using it